src/components/ADempiere/DataTable/dataTables-Script.js

```diff
@@ -723,34 +689,24 @@ export default {
           sums[index] = ''
           return
         }
-        const values = this.getDataSelection.map(item => Number(item[columnItem.property]))
-        if (values.every(value => isNaN(value))) {
-          sums[index] = 0
-        } else {
-          const total = values.reduce((prev, curr) => {
-            const value = Number(curr)
-            if (!isNaN(value)) {
-              return prev + curr
-            }
-            return prev
-          }, 0)
-          sums[index] = this.formatNumber({
-            displayType,
-            number: total
-          })
-        }
+
+        const values = selectionData.map(item => Number(item[columnItem.property]))
```

Comment:
Don't touch it if not exists changes

Reply:
The change is around the format of the summation totals table.